docs/openapi.yml

```diff
@@ -397,3 +398,214 @@ paths:
+
+  /switcher/get_shutter_state:
+    get:
+      description: Get state of the Shutter device.
+      tags:
+        - "API Endpoints"
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: the id of the selected device
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      responses:
+        "200":
+          description: A JSON dictionary of breeze state
+          content:
+            application/json:
+              examples:
+                direction: "SHUTTER_STOP"
+                position: 95
+        "500":
+          description: A JSON object hinting for the error
+          content:
+            application/json:
+              examples:
+                "Example for error when failed to get the current device state":
+                  value:
+                    error: "an error of some type was raised"
+
+  /switcher/set_stop_shutter:
+      post:
+      description: Stop shutter
+      tags:
+        - "API Endpoints"
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: the id of the selected device
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      responses:
+        "200":
+          description: An empty JSON object
+        "500":
+          description: A JSON object hinting for the error
+          content:
+            application/json:
+              examples:
+                "Example for error when failed to turn off the device":
+                  value:
+                    error: "an error of some type was raised"
+
+  /switcher/control_breeze_device:
+    post:
+      description: Use for update breez device state.
+      tags:
+        - "API Endpoints"
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: the id of the selected device
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+            example:
```

Comment:
we're missing the example title as the key for the example object.

we also need multiple examples here, other then the `remote_id`, all other properties are not mandatory, you can send a request using just one of them, as well as all of them, we need the examples to relay this.